opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -237,8 +243,27 @@ def __exit__(
         exc_tb: typing.Optional[python_types.TracebackType],
     ) -> None:
         """Ends context manager and calls `end` on the `Span`."""
+
         self.end()
 
+        if (
+            exc_val is not None and
+            self.status is None and
+            self._auto_update_status
+        ):
+            if isinstance(exc_val, Status):
+                self.set_status(exc_val.status)
+
+            else:
+                self.set_status(UnknownStatus)
+
+            # return False
+        # return True
+
+        # FIXME https://docs.python.org/3.7/reference/datamodel.html?highlight=__exit__#object.__exit__
+        # says that a boolean value should be returned. Is the lack of the
+        # return of a boolean value here intentional?
+
```

Comment:
Yes, it is intentional. We get pylint problems otherwise and it is completely idiomatic to not use a return statement in context managers that never swallow exceptions, like this.

Reply:
Thanks, removed the `FIXME`.